docs/recipes/README.md

```diff
@@ -44,13 +44,13 @@ function* watchInput() {
 }
 ```
 
-In the above example `handleInput` waits for 500ms before performing its logic. If the user types something during this period we'll get more `INPUT_CHANGED` actions. Since `handleInput` will still be blocked in the `delay` call, it'll be cancelled by `watchInput` before it can start performing its logic.
+The `delay` function implements a simple debounce using a Promise. In the above example `handleInput` waits for 500ms before performing its logic. If the user types something during this period we'll get more `INPUT_CHANGED` actions. Since `handleInput` will still be blocked in the `delay` call, it'll be cancelled by `watchInput` before it can start performing its logic.
```

Comment:
You could put its implementation here between the lines, so people dont think its some kind of magic, but just a simple function returning a promise.